test/AutoRest.TestServer.Tests/Infrastructure/TestServerV2.cs

```diff
@@ -26,29 +26,26 @@ public TestServerV2()
             var wiremock = Path.Combine(nodeModules, "wiremock", "jdeploy-bundle");
             var wiremockJar = Directory.GetFiles(wiremock, "*.jar").Single();
             var root = GetBaseDirectory();
-
-            var processStartInfo = new ProcessStartInfo("java", $"-jar {wiremockJar} --root-dir {root} --port 0");
-            // Use random port
-            processStartInfo.Environment["PORT"] = "0";
-            processStartInfo.RedirectStandardOutput = true;
-            processStartInfo.RedirectStandardError = true;
+            var port = new Random().Next(10000, 65536).ToString();
```

Comment:
We are skipping all V2 tests anyway so let's not to the port change.

Reply:
The V2 tests will not be skipped once the newest test server version is used. So, this will allow that ignore to be removed as soon as we update it. No point in removing it, and it made sense to do the same change in both places at once.

Reply:
I don't see what was fixed in the linked PR to make the V2 work

Reply:
From what I'm told, the recordings now do not use base64 encoded strings. They use regular strings. This allows the utility method that would inject the correct port to work on the recordings. Obviously, I haven't tried it but I'm trusting Laurent on it.

Reply:
We won't be merging this kind or port generation anyway. It would make our tests very flaky.

Reply:
The port was random to begin with. We were just reading it from the test server instead of sending it. Unless I'm misunderstanding the code that was already there.

Reply:
When server asks for random port from the OS its guaranteed to get a free one when you are generating a random number there are no guarantees so the server startup might fail and it happens more often than you think considering how often we restart processes.

Reply:
Any chance we could simply make that same guarantee here? I'd really like having working paging tests so, sometime in the near future, implement the rest of the features for paging, with the assurance it passes the tests. If we have to wait for the test server change, this PR will have to keep waiting. And, I have my big rename PR based on this one.

Reply:
Nope, there is an inherent race there. We can merge test server PR and update it very quickly.

Reply:
And for V2 we don't even need the change.

Reply:
Testserver PR is in.

Reply:
The test server auth cred was expired, so no packages were being published. For now, Laurent has used a personal credential for it. After publishing 2.10.5, we found issues with the paging tests. He fixed them and published 2.10.6. Now, I was able to run some of the tests on V2.